scripts/format_batch_with_env.sh

```diff
@@ -185,7 +187,8 @@ if [ "$project" = "workers" ]; then
                 echo "Made $FILEPATH"
             done
             echo "Made $output_dir/$OUTPUT_FILE"
-        elif [ "$OUTPUT_FILE" = "smasher.json" ] || [ "$OUTPUT_FILE" = "create_qn_target.json" ] || [ "$OUTPUT_FILE" = "create_compendia.json" ] || [ "$OUTPUT_FILE" = "create_quantpendia.json" ] || [ "$OUTPUT_FILE" = "tximport.json" ]; then
+        # From https://unix.stackexchange.com/a/111517
+        elif (echo "$NO_RAM_JOB_FILES" | grep -Fqw "$OUTPUT_FILE"); then
```

Comment:
It's moments like this when I most wish for switching to python like in resources-portal

Reply:
Absolutely. I've started putting new scripts in python. Eventually maybe we'll even take the time to rewrite some of the old bash ones.